### ISSUE TITLE
Quality controle on "Organisations using Fastify"

### ISSUE THREAD

**Issue by Uzlopak** [fastify/website#249]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the issue has not already been raised

### Issue

I find it critical, that we accept every logo and organization without due dilligence.

**Comment by mcollina** [comment]
I think so too, we should really make this stricter.

@Eomm @jsumners wdyt?

Should we remove it completely?

**Comment by Eomm** [comment]
Let's keep the sponsors only

**Comment by jsumners** [comment]
The only framework I can find that even lists any users is Hapi. And they only list them with testimonials on https://hapi.dev.

I think the logos are good to show that people are using the framework, but agree there might need to be some more judicious addition of them.

I would be more in favor of removing them completely than only listing the logos of sponsors. We never advertised that we would promote the logos of sponsors, and I think it would be too preferential to drop all of the others and not those.

To summarize, my preferences would be:

1. Drop them all.
2. Incorporate a few "big name" logos with testimonials.

**Comment by Eomm** [comment]
> Drop them all.

This clashes a bit with our Tiers: https://opencollective.com/fastify
This use case is not covered by our policy 😭

**Comment by Uzlopak** [comment]
Tbh i wanted to report this issue for months now and was not sure how you would take it. I have two worries:
1. We potentially use the logos of companies who dont use fastify and maybe worse: never agreed to use their logo on our page. 
2. We potentially make advertisement for some shady websites, for free. 

Free Use and royalty free is some concept of USA, which is not feasible in EU.  
And for Nr. 2 we never verify the claim and maybe tomorrow some adult entertainment or gambling or crypto wants to be advertised. I fear crypto is already part of our list.

**Comment by jsumners** [comment]
> > Drop them all.
> 
> 
> 
> This clashes a bit with our Tiers: https://opencollective.com/fastify
> 
> This use case is not covered by our policy 😭 

Ugh. I forgot that exists. But it doesn't change my opinion.

**Comment by jsumners** [comment]
> Tbh i wanted to report this issue for months now and was not sure how you would take it. I have two worries:
> 
> 1. We potentially use the logos of companies who dont use fastify and maybe worse: never agreed to use their logo on our page. 
> 
> 2. We potentially make advertisement for some shady websites, for free. 
> 
> 
> 
> Free Use and royalty free is some concept of USA, which is not feasible in EU.  
> 
> And for Nr. 2 we never verify the claim and maybe tomorrow some adult entertainment or gambling or crypto wants to be advertised. I fear crypto is already part of our list.

I'm not really following this.

**Comment by Eomm** [comment]
@fastify/collaborators Please share your vote here: https://github.com/orgs/fastify/discussions/48

**Comment by Eomm** [comment]
Superseded by https://github.com/fastify/website/issues/255
